src/CMakeLists.txt

```
@@ -58,8 +58,13 @@ cmd_option( use_ladspa "Use LADSPA plug-in support [ON, OFF]" ON )
 set( USE_LADSPA ${use_ladspa} CACHE INTERNAL "" )
 
 # Handle VST option
-cmd_option( use_vst "Use VST2 plug-in support [ON, OFF]" ON )
-set( USE_VST ${use_vst} CACHE INTERNAL "" )
+if( NOT CMAKE_SYSTEM_NAME MATCHES "Darwin|Windows" )
+  pkg_check_modules(GDK3_X11 gdk-x11-3.0)
+  cmake_dependent_option( use_vst "Use VST2 plug-in support [ON, OFF]" ON "GDK3_X11_FOUND" OFF)
+  set( USE_VST ${use_vst} CACHE INTERNAL "" )
```

```suggestion
  cmake_dependent_option( USE_VST "Use VST2 plug-in support [ON, OFF]" ON "GDK3_X11_FOUND" OFF)
```
I don't think the intermediate lowercase variable is needed.